### ISSUE TITLE
undefined address when using braavos

### ISSUE THREAD

**Issue by Th0rgal** [apibara/starknet-react#158]
Hey there,

We noticed an issue happening with Braavos (it worked fine with argent-x). After reloading the page, the multicall signing panel from braavos doesn't open when clicking on the button which calls the execute. The 'error' printed was ``Cannot read properties of undefined (reading 'address')``. Braavos team told us this could be because the connector was not injected properly (we hardcoded it) or there is a problem in the starknet-execute. We tried to log the account in hooks/execute.js and it was undefined which is strange because window.starknet_braavos was defined. When changing the account value to window.starknet_braavos, it seems to work correctly.

Have a great day,
Thomas

**Comment by avimak** [comment]
1. @Th0rgal team is using `@starknet-react/core` "all in", e.g. not connecting directly with `get-starknet` or accessing the window injected object directly
2. they get the info from other hooks correctly (e.g. connected account's address, etc.)
3. when they call `useStarknetExecute` - while connected, with a valid `callData` object - the hook crashes at line 54 -
```
                    const account = await connector.account();
                    if (account.address === accountAddress) { -> `account` is undefined here, although window.starknet_braavos is valid + connected + have all the required information listed correctly
```

**Comment by avimak** [comment]
hey @fracek, any updates on this one?

**Comment by fracek** [comment]
Sorry, this was fixed in #159. Basically braavos was installed but not connected and so `account` was undefined and the code tried to access its `address` property.

**Comment by avimak** [comment]
@fracek but why did it happen only with Braavos? all starknet wallets have an undefined `account` value while disconnected?
